server/app/graphql/types/queries/drug_lookup_query.rb

```diff
@@ -2,20 +2,20 @@ module Types::Queries
   module DrugLookupQuery
     def self.included(klass)
       klass.field :drug_matches, Types::DrugMatchType, null: false do
-        description "Match Drug search terms to known drugs in the database."
+        description "Case-insensitively match Drug search terms to known drugs in the database."
         argument :search_terms, [GraphQL::Types::String], required: true
       end
 
       def drug_matches(search_terms: )
-        remaining_terms = Set.new(search_terms)
+        remaining_terms = Set.new(search_terms.map(&:upcase))
 
         results = {
           direct_matches: [],
           ambiguous_matches: [],
           no_matches: []
         }
 
-        #find exact matches on drug name 
+        #find exact matches on drug name
         direct_symbol_matches = Drug.where(name: remaining_terms)
```

Comment:
Can drug names ever be mixed case in the DB?

Reply:
I had thought we enforced this constraint in the base importer method or the model, but it looks like we just upcase in each importer's call to create_drug_claim (this is sort of scary but apparently it's been working -- just checked `SELECT * FROM drugs WHERE upper(name) != name;` and there were no results)